config/webpack/buildLoaders.ts

```diff
@@ -0,0 +1,45 @@
+import { ModuleOptions } from "webpack";
+import { BuildOptions } from "./types/types";
+import MiniCssExtractPlugin from "mini-css-extract-plugin";
```

Comment:
It's just a nit, but let's keep imports from 3rd parties before internal imports 
Here
import { ModuleOptions } from "webpack";
import MiniCssExtractPlugin from "mini-css-extract-plugin";

and then 

import { BuildOptions } from "./types/types";